docs/DeveloperGuide.md

```diff
@@ -154,6 +157,37 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### View Feature
+
+### Current Implementation
+
+The view function is facilitated by `ViewXCommand` (`X` is a placeholder for the specific command name e.g., `ViewCustomerCommand`).
+
+Here X can be appointment/customer/service/technician/vehicle.
+
+The Sequence Diagram below illustrates the interactions within the Logic component for the execute("viewcustomer 1") API call.
+
+<img src="images/ViewSequenceDiagram.png" width="600"/>
```

Comment:
Do remember to check if the size of the diagram is big enough to be displayed clearly, think 600 may be too small, couldn't really see the labels when I looked at the md file from github

Reply:
ops